### ISSUE TITLE
"qsub -q special -I" with walltime >72h doesn't work

### ISSUE THREAD

**Issue by boegel** [hpcugent/slurm-torque-wrappers#24]
```
$ qsub --debug -q special -I -l walltime=100:0:0
DEBUG: Generated interactive  command '/usr/bin/salloc -J INTERACTIVE --nodes=1 --time=6000 /usr/bin/srun --pty --mem-per-cpu=0 --partition victini_special --chdir=/user/gent/400/vsc40023 --cpu-bind=v,none --export=USER,HOME,TERM /usr/bin/bash -i -l'
salloc: error: Job submit/allocate failed: Requested time limit is invalid (missing or exceeds some limit
```

The problem is that the `--partition victini_special` should be passed to `salloc` rather than `srun`.

Question is, do we actually *want* to support interactive jobs longer than 72h?

**Comment by stdweird** [comment]
partition handling should be added to https://github.com/hpcugent/slurm-torque-wrappers/blob/master/files/qsub.pl#L662

**Comment by boegel** [comment]
fixed in #25
